src/go/MONIT/monit.go

```diff
@@ -799,14 +879,21 @@ func main() {
 		log.SetFlags(log.LstdFlags)
 	}
 	var e error
-	DataSources = datasources(defaultUrl, token, verbose)
```

Comment:
@mrceyhun it is not correct logic here, we should support both reading datasource with admin token and bypass reading from github and reading from github repo. I suggest the following changes:
- make datasources function to return both datasources and error
- if error is nil then you don't need to readDS, otherwise you will use readDS
Doing this way will accomplish all use-cases:
- users will be able to read from github url
- admins can by pass github url and read datasoruce with admin token.